app.ts

```diff
@@ -41,6 +55,35 @@ export const getManifestFileName = (
   return "Clarinet.toml";
 };
 
+export const tryParseRemoteDataSettings = (
+  manifestPath: string,
+  radio: EventEmitter
+): RemoteDataSettings => {
+  const clarinetToml = toml.parse(readFileSync(resolve(manifestPath), "utf-8"));
+  const remoteDataUserSettings =
+    clarinetToml.repl?.["remote_data"] ?? undefined;
+
+  if (
+    remoteDataUserSettings !== undefined &&
+    (!remoteDataUserSettings?.["api_url"] ||
+      !remoteDataUserSettings?.["enabled"] ||
+      !remoteDataUserSettings?.["initial_height"])
+  ) {
+    throw new Error(invalidRemoteDataErrorMessage);
+  }
+
+  if (remoteDataUserSettings) {
+    radio.emit(
+      "logMessage",
+      yellow(
+        "\nUsing mainnet data. Setting up the environment can take up to a minute..."
```

Comment:
```suggestion
        "\nUsing remote data. Setting up the environment can take up to a minute..."
```

Is it always mainnet? Can it be also some prelaunch, tester or other live network? Perhaps _remote_ is a better name?

Reply:
Great catch! 🚀